Jenkinsfile

```diff
@@ -69,30 +112,37 @@ pipeline {
                                 retry(3) {
                                     checkout scm
                                 }
-                                withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
+                                withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '-Xmx8192M -Xss128M -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC -Djava.security.egd=file:/dev/./urandom') {
                                     withCredentials([string(credentialsId: 'sonarqube-token', variable: 'SONAR_TOKEN')]) {
                                         sh 'mvn -q -B -Dfindbugs.skip=true -Dcheckstyle.skip=true org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Dsonar.host.url=https://sonarqube.com -Dsonar.login=$SONAR_TOKEN  -Dsonar.organization=codice -Dsonar.projectKey=ddf -pl !$DOCS,!$ITESTS'
                                     }
                                 }
                             }
                         },
+                        // Coverity will be skipped on all PR builds
                         coverity: {
                             node('linux-medium') {
-                                retry(3) {
+                              script {
+                                if (env.CHANGE_ID != null) {
+                                  echo "Skippng Coverity on PR build"
+                                } else {
+                                  retry(3) {
                                     checkout scm
+                                  }
+                                  withMaven(maven: 'M35', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '-Djava.security.egd=file:/dev/./urandom') {
```

Comment:
✏️  These files are deeply indented and fairly difficult to read. We may want to make sure our indentations are consistent throughout. That might help a little.

Reply:
I probably need to run them through a formatter

Reply:
should be fixed now